programmers/난이도별/level02.기능개발/sjjyy.java

```diff
@@ -0,0 +1,84 @@
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.Queue;
+
+public class sjjyy {
+    public static int[] solution(int[] progresses, int[] speeds) {
+        int len = progresses.length;
+        Queue<Integer> queue = new LinkedList<>();
+        int index = 0, count = 0;
+
+        while (index < len)
+        {
+            for (int i = 0; i < len; i++)
+                progresses[i] += speeds[i]; // 하루 작업 진도
+
+            if (progresses[index] >= 100) // 작업 배포가 가능할 때
+            {
+                while (index < len && progresses[index] >= 100) // 같이 배포가 가능한 개수
+                {
+                    count++;
+                    index++;
+                }
+
+                queue.offer(count); // 배포된 개수 저장
+                count = 0;
+            }
```

Comment:
```suggestion
            if (progresses[index] < 100) continue;
            // 작업 배포가 가능할 때
            while (index < len && progresses[index] >= 100) // 같이 배포가 가능한 개수
            {
                count++;
                index++;
            }

            queue.offer(count); // 배포된 개수 저장
            count = 0;
```
if로 전체 로직을 감싸지 않도록 해주세요!

Reply:
항상 감사합니다!